src/lakefs_spec/util.py

```diff
@@ -17,7 +21,23 @@ class PaginatedApiResponse(Protocol):
 def depaginate(
     api: Callable[..., PaginatedApiResponse], *args: Any, **kwargs: Any
 ) -> Generator[Any, None, None]:
-    """Send a number of lakeFS API response documents to a generator."""
+    """
+    Send a number of received lakeFS API response documents to a generator.
```

Comment:
I was taking over the "send/receive" idiom by Python generators used in this SO answer about `yield from`: https://stackoverflow.com/questions/9708902/in-practice-what-are-the-main-uses-for-the-yield-from-syntax-in-python-3-3

But I guess `unwrap` also can work since the return type is a list.

Reply:
The argument makes sense, but since `resp.results` is a list, it doesn't establish the bidirectional binding between generators mentioned in the answer. (plus, I find that an implementation detail not really relevant for the user - in the end, all they care for is the results, not how we got them out of the API)

Reply:
Fixed in https://github.com/aai-institute/lakefs-spec/commit/26f4b92746c4d4dd473f1f074801c7d881148569.